eyey28/demo-locator
# Added Demo Locator Feature

**Description**

Added the Demo Locator Feature.

**Validation**

- Open the demo-locator folder.
- Open the index.html on any browser. (This is developed in Chrome browser. Not yet tested on other browsers.)
- The HTML, CSS, and JS should work properly.

 Desktop Image:
![desktop-image](https://github.com/eyey28/demo-locator/assets/19793538/e01175e9-081a-4af5-81ce-794135b6782f)

Tablet Image:
![tablet-image](https://github.com/eyey28/demo-locator/assets/19793538/e77a6013-a56a-46ed-9430-376b21640cbe)

Mobile Image:
![mobile-image](https://github.com/eyey28/demo-locator/assets/19793538/e0d6b08a-171d-48fc-bbd3-2050ffaba794)
